redis/Cargo.toml

```diff
@@ -10,8 +10,36 @@ repository = "https://github.com/djc/bb8"
 [dependencies]
 async-trait = "0.1"
 bb8 = { version = "0.8", path = "../bb8" }
-redis = { version = "0.24", default-features = false, features = ["tokio-comp"] }
+redis = { version = "0.24" }
```

Comment:
We'll still want the `default-features = false` here, I think, as the upstream crate's `default` features will then be re-enabled by our forwarding of our `default` feature to `redis/default`. I forget how or why redis has the `tokio-comp` feature but I also feel like we should keep that enabled unconditionally.

Reply:
Change made